harness.sh

```diff
@@ -280,14 +292,104 @@ tmux send-keys "./ctl createnewaccount c2" C-m
 tmux send-keys "./ctl getnewaddress c2" C-m
 tmux send-keys "./ctl getnewaddress default" C-m
 tmux send-keys "./ctl getbalance"
+
+if [ "${NETWORK}" = "simnet" ]; then
+################################################################################
+# Setup the voting node.
+################################################################################
+cat > "${NODES_ROOT}/vnode/ctl" <<EOF
+#!/bin/zsh
+dcrctl -C ../dcrvctl.conf \$*
+EOF
+chmod +x "${NODES_ROOT}/vnode/ctl"
+
+tmux new-window -t $SESSION:4 -n 'vnode'
+tmux send-keys "cd ${NODES_ROOT}/vnode" C-m
+
+echo "Starting ${NETWORK} voting node"
+
+tmux send-keys "dcrd -C ${DCRD_CONF} \
+--rpccert=${DCRD_RPC_CERT} --rpckey=${DCRD_RPC_KEY} \
+--rpcuser=${RPC_USER} --rpcpass=${RPC_PASS} \
+--connect=127.0.0.1:18555 \
+--listen=127.0.0.1:19559 --rpclisten=127.0.0.1:19560 \
+--miningaddr=${CPU_MINING_ADDR} \
+--appdata=${NODES_ROOT}/vnode \
+--txindex \
+--debuglevel=info \
+--simnet" C-m
+
+################################################################################
+# Setup the voting node's dcrctl (vctl).
+################################################################################
+sleep 3
+cat > "${NODES_ROOT}/vnode/mine" <<EOF
+#!/bin/sh
+  NUM=1
+  case \$1 in
+      ''|*[!0-9]*)  ;;
+      *) NUM=\$1 ;;
+  esac
+  for i in \$(seq \$NUM) ; do
+    dcrctl -C ../dcrvctl.conf generate 1
+    sleep 0.5
+  done
+EOF
+chmod +x "${NODES_ROOT}/vnode/mine"
+
+tmux new-window -t $SESSION:5 -n 'vctl'
+tmux send-keys "cd ${NODES_ROOT}/vnode" C-m
+
+# mine some blocks to start the chain if the active network is simnet.
+tmux send-keys "./mine 30" C-m
+sleep 5
+echo "Mined 30 blocks, at stake enabled height (SEH)"
+
+################################################################################
+# Setup the voting wallet.
+################################################################################
+cat > "${NODES_ROOT}/vwallet/ctl" <<EOF
+#!/bin/sh
+dcrctl -C ../dcrvwctl.conf --wallet \$*
+EOF
+chmod +x "${NODES_ROOT}/vwallet/ctl"
+
+cat > "${NODES_ROOT}/vwallet/tickets" <<EOF
+#!/bin/sh
+NUM=1
+case \$1 in
+    ''|*[!0-9]*) ;;
+    *) NUM=\$1 ;;
+esac
+./ctl purchaseticket default 999999 1 \`./ctl getnewaddress\` \$NUM
+EOF
+chmod +x "${NODES_ROOT}/vwallet/tickets"
+
+tmux new-window -t $SESSION:6 -n 'vwallet'
+tmux send-keys "cd ${NODES_ROOT}/vwallet" C-m
+tmux send-keys "dcrwallet -C vwallet.conf --create" C-m
+echo "Creating ${NETWORK} voting wallet"
+sleep 1
+tmux send-keys "${WALLET_PASS}" C-m "${WALLET_PASS}" C-m "n" C-m "y" C-m
+sleep 1
+tmux send-keys "${VOTING_WALLET_SEED}" C-m C-m
+tmux send-keys "dcrwallet -C vwallet.conf --debuglevel=debug" C-m
+
+################################################################################
+# Setup the voting wallet's dcrctl (vwctl).
+################################################################################
+sleep 5
```

Comment:
This sleep may need to be slightly longer. The line `./tickets 300` in tmux window 7 is giving me occassional errors because the dcrwallet in window 6 is not yet RPC ready.

When I increase `sleep 5` to `sleep 10` the issue does not occur.